install_deps.sh

```diff
@@ -363,6 +363,23 @@ git clone -b v0.7.0 --depth 1 https://github.com/jaegertracing/jaeger-client-cpp
     cd ${HOME} && \
     rm -r jaeger-client-cpp
 
+# ccache is used to accelerate C/C++ recompilation
+CCACHE_VER=4.6.1
+cd "$HOME"
+wget https://github.com/ccache/ccache/releases/download/v${CCACHE_VER}/ccache-${CCACHE_VER}.tar.xz && \
+  tar xvf ccache-${CCACHE_VER}.tar.xz && \
+  cd ccache-${CCACHE_VER} && \
+  mkdir build && cd build && \
+  cmake -DCMAKE_BUILD_TYPE=Release .. && make && make install && \
+  cd "${HOME}" && \
+  rm -rf ccache-${CCACHE_VER} ccache-${CCACHE_VER}.tar.xz
+CCACHE_PATH=$(which ccache)
+ln -s "${CCACHE_PATH}" /usr/local/bin/gcc
```

Comment:
do you have to create these links manually? Isn't there a ccache script that does it?

Reply:
CMAKE_CXX_COMPILER_LAUNCHER/CMAKE_C_COMPILER_LAUNCHER might crate the links but then the image is removed and the linsk too, so if we want permanent links we must do this as part of the image.
Still, without ovverriding the path nothing will work, so it's ok to put the links here even if ccache is disabled.